### PR TITLE
add css variables

### PR DESCRIPTION
**Pull request by AnkePfei2fer**

### CODE REVIEW

File: src/global.css [hunk 1]
```diff
@@ -1,5 +1,20 @@
 @import url('https://fonts.googleapis.com/css2?family=Signika&display=swap');
 
+:root {
+  --font-family: "Signika", sans-serif;
+  --color-primary: #91c788;
+  --color-secondary: #ff8473;
+  --color-secondary-light: #FFC0B8;
+  --color: #7C7C7C;
```
Comment (manusmd) [inline src/global.css]: maybe a more specific name for the variable?

**Review by manusmd — COMMENTED** [review]
nice work! Just a small thing to check

File: src/global.css (hunk 1 above)
Comment (AnkePfei2fer) [reply src/global.css]: thx! Which variable do you mean? color?

Comment (manusmd) [reply src/global.css]: yes :)

**Pull request merged**